UPLOADING.md

```diff
@@ -65,10 +65,31 @@ At this point, there should be **no further changes in your code**.
 
 Run the following command **in a folder that does not include your data-loading script**:
 
+Test both the original dataset schema/config and the bigbio schema/config. 
+
+```python
+from datasets import load_dataset
+
+dataset_orig = load_dataset("bigscience-biomedical/<your_dataset_name>", name="source", use_auth_token=True)
```

Comment:
@galtay Can you provide an example/context at which users would need to pull the dataset under different names?

Reply:
`name` is how you specify the particular dataset loader config you want to use. for the hack-a-thon we ask participants to implement the original dataset schema (i.e. as close as we can get to the native dataset format) as a `source` config and then to also implement the `bigbio` config for the `bigbio` schema relevant to the dataset task (e.g. `kb`, `translation`, ...) . All of the implementations in the `examples` folder should do this ... this one is probably easiest to view at the moment https://github.com/bigscience-workshop/biomedical/blob/master/examples/bc5cdr.py#L80

Reply:
im going to add this because its easier to make edits on top of what's here - i approve these changes because the authtoken and the name of the view are useful commands to default to. we will need to add more info around how to access views etc and what they might mean but fine for now